chapter04/dcox/ex4_3

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+sub total {
+  my $sum = 0;
+
+  $sum += $_ foreach(@_);
+
+  $sum;
+}
+
+sub average {
+  my $avg = 0;
+
+  $avg = &total(@_)/(scalar @_) if(scalar @_);
```

Comment:
Again, a nice concise way of doing things, I'm not sure if you need to use `scalar` though